apps/api/src/resources/swagger.yaml

```diff
@@ -638,6 +670,100 @@ components:
         applicationId:
           description: The id of the Application
           type: number
+    RequestRevisions:
+      type: object
+      required:
+        - applicationId
+        - role
+        - repId
```

Comment:
Sorry just noticed there's 2 instances of `repId` in Swagger and 1 instance in application-router